### ISSUE TITLE
Feature Request: FirstOrNew, FirstOrCreate

### ISSUE THREAD

**Issue by nickaguilarh** [baopham/laravel-dynamodb#178]
Is there any plan or roadmap for supporting first or new / first or create? 
It would be really nice if we had those methods.

**Comment by baopham** [comment]
Yep, this is easy to do. If anyone is interested, feel free to make a PR. I won't work on this until next weekend.

**Comment by nickaguilarh** [comment]
I just made the PR, let me now if something else is needed.
https://github.com/baopham/laravel-dynamodb/pull/179
